app/controllers/chefs/ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Chefs::IngredientsController < ApplicationController
+  def index
+    @chef = Chef.find(params[:chef_id])
+    @ingredients = @chef.ingredients
```

Comment:
Great controller organization! Keep in mind, too, that you don't strictly need the @ingredients var as you can call @chef.ingredients in the view too